CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 ## Unreleased
 
 ### Fixed
-
+- [#6834](https://github.com/thanos-io/thanos/pull/6834) Replaces instances of io/ioutil with io and os in Go code
```

Comment:
We don't need changelog for this as it is not a user facing change.

Reply:
Sounds good, one moment